src/main/java/be/polyscripts/contactmanagerapp/dto/AuthenticationRequest.java

```diff
@@ -0,0 +1,35 @@
+package be.polyscripts.contactmanagerapp.dto;
+
+public class AuthenticationRequest {
```

Comment:
use lombok @Data to generate setters and getters

Reply:
done !